client/src/index.js

```diff
@@ -2,42 +2,37 @@ import React from "react";
 import ReactDOM from "react-dom";
 import "semantic-ui-css/semantic.min.css";
 import "./index.css";
-import { BrowserRouter as Router, Link, Route, Switch } from "react-router-dom";
-import { Grid, Container } from "semantic-ui-react";
+import { BrowserRouter as Router, Route } from "react-router-dom";
+import { Container } from "semantic-ui-react";
 import Home from "./components/Home";
 import About from "./components/About";
 import Status from "./components/Status";
 import Login from "./components/Login";
 import NavBar from "./components/NavBar";
+import ApplicantRegister from "./components/ApplicantRegister";
 import NewOpportunityForm from "./components/NewOpportunityForm";
+import MainRegister from "./components/MainRegister";
 import ApplicantProfile from "./components/ApplicantProfile";
 
 const Routes = () => {
   return (
     <Container>
       <Router>
         <NavBar />
-        <Grid width={15} centered>
-          <div style={{ margin: "20px" }}>
-            <Switch>
-              <Route path="/" exact component={Home} />
-              <Route path="/about" exact component={About} />
-              <Route path="/status" exact component={Status} />
-
-              <Route path="/login" exact component={Login} />
-              <Route
-                path="/applicant-profile"
-                exact
-                component={ApplicantProfile}
-              />
-              <Route
-                Path="/create-opportunity"
-                exact
-                component={NewOpportunityForm}
-              />
-            </Switch>
-          </div>
-        </Grid>
+        <div style={{ margin: "20px" }}>
+          <Route path="/" exact component={Home} />
+          <Route path="/about" component={About} />
+          <Route path="/status" component={Status} />
+          <Route path="/create-opportunity" component={NewOpportunityForm} />
+          <Route path="/main-register" component={MainRegister} />
+          <Route
+            path="/applicant-register"
+            exact
+            component={ApplicantRegister}
+          />
+          <Route path="/applicant-profile" exact component={ApplicantProfile} />
+          <Route path="/login" exact component={Login} />
```

Comment:
do we need `exact` on these? What if we need to add sub-routes/params to it in the future?

I'm assuming it will work even if you remove it?